dags/hindawi/hindawi_file_processing.py

```diff
@@ -0,0 +1,14 @@
+import airflow
+from airflow.decorators import dag, task
+
+
+@dag(start_date=airflow.utils.dates.days_ago(0))
+def hindawi_file_processing():
+    @task()
+    def parse(**kwargs):
+        if "params" in kwargs and "article" in kwargs["params"]:
+            record = kwargs.get("params").get("record")
```

Comment:
this will fail you need something like `kwargs.get("params", {}).get("record")`

Reply:
but if I'm already checking is the params is in kwargs, why we need the default values?

Reply:
you need a default value for `params` because if params is not in `kwargs` it will throw an exception.

Reply:
but it will never reach this line below, if the 'prams' is not there, the whole function will return null

Reply:
you don't need the `if` with `.get(params, {})`